docs/source/reference/auto-stop.rst

```diff
@@ -41,6 +41,12 @@ To schedule autodown for a cluster, pass the ``--down`` flag to either :code:`sk
    # Or:
    sky autostop mycluster2 -i 10 --down
 
+.. note::
+
+  The autostop/autodown logic will be automatically executed by the remote
+  cluster.  Your local machine does *not* need to stay up for them to take
```

Comment:
```suggestion
  cluster. Your local machine does *not* need to stay up for them to take
```
nit

Reply:
This is old school convention :) https://en.wikipedia.org/wiki/Sentence_spacing